src/pages/PlaylistVideos/PlaylistVideos.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import Navbar from "../../components/Navbar/Navbar";
```

Comment:
You can try barrel imports if you need to import many files. That would decrease the number of lines of code.

Reply:
Sure!